client/src/components/layout/EditForm.js

```diff
@@ -0,0 +1,190 @@
+import React, { useState, useEffect } from "react"
+import { Redirect } from "react-router-dom"
+
+import translateServerErrors from "../../services/translateServerErrors.js"
+import ErrorList from "./ErrorList.js"
+
+const EditForm = (props) => {
+  const [form, setForm] = useState({
+    title: "",
+    continent: "",
+    country: "",
+    city: "",
+    numberOfDays: "",
+    description: "",
+    userId: "",
+    id: ""
+  })
+  const [shouldRedirect, setShouldRedirect] = useState(false)
+  const [errors, setErrors] = useState([])
```

Comment:
sorry, can you triple check that your errors work correctly? generally errors is an object, but you may have a different implementation, so no worries otherwise